data-prepper-core/src/integrationTest/java/org/opensearch/dataprepper/plugins/BasicEventsTrackingTestProcessor.java

```diff
@@ -20,6 +20,6 @@ public class BasicEventsTrackingTestProcessor extends BaseEventsTrackingProcesso
     private static final String PLUGIN_NAME = "basic_events_tracking_test";
 
     public BasicEventsTrackingTestProcessor() {
-        super(PLUGIN_NAME, PROCESSED_EVENTS_MAP);
+        super(PLUGIN_NAME, PROCESSED_EVENTS_MAP, -1);
```

Comment:
Why  -1?

Reply:
We don't track it for this processor. I made this a static final with a name to add clarity.